impl/src/main/java/slack/ThreadMessageHandler.java

```diff
@@ -144,11 +145,12 @@ private Auth as(Long userId) {
 
                     return answerResource.answerQuestion(as(user.getId()), answer, question.getId()).ignoreElements().cast(Void.class);
                 }
-        ));
+            ));
     }
 
     /**
-     * TODO figure out how to create a title from the message
+     * Gets the title from the message.
+     *
      * @param message the message from slack
      * @return the title
```

Comment:
No need for double descriptions like "Gets the title" and "@return The title", just a @return is good enough when it describes the purpose :)

Reply:
fixed :)